tokio-util/Cargo.toml

```diff
@@ -30,6 +30,7 @@ time = ["tokio/time","slab"]
 io = []
 io-util = ["io", "tokio/rt", "tokio/io-util"]
 rt = ["tokio/rt", "tokio/sync", "futures-util", "hashbrown"]
+lrtd = []
```

Comment:
Please just keep the existing `--cfg tokio_taskdump` flag rather than introduce a new one.

Reply:
@Darksonn  I have separated the blocking detection from threads state dump in the latest version. The detection implementation is architecture and os agnostic,  So in theory one using the LRTD blocking detector can plug in arch/env specific state dump implementations. So that I don't have to relocate this multiple times, where is the best place to add lrtd.rs and it's tests?